ui/tests/acceptance/secrets/backend/ssh/role-test.js

```diff
@@ -12,11 +12,12 @@ module('Acceptance | secrets/ssh', function (hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(function () {
+    this.timestamp = new Date().getTime();
     return authPage.login();
   });
 
-  const mountAndNav = async () => {
-    const path = `ssh-${new Date().getTime()}`;
+  const mountAndNav = async (assert, uid) => {
```

Comment:
out of curiosity - why does `assert` need to be passed here? I see it's passed below, but I don't see it used in the method itself?

Reply:
Good catch, I think it was leftover from a time when we were asserting within this block. I can remove it